components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -110,8 +110,8 @@
               (click)="saveServer()">
               <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
               <span *ngIf="saveInProgress">Saving...</span>
-              <span *ngIf="!saveInProgress">Save</span></chef-button>
-            <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
```

Comment:
@jong-progress - My gut reaction was to flag this, because we want to make sure we notify the users that all of the changes have been saved.  _However_, we now display a ribbon across the top after saving that says the changes have been updated successfully.  Are we ok to remove this line of code now?

What's being removed here is the tag that says "All changes saved" next to the save button when a chef server name has been updated.

![Screen Shot 2021-04-26 at 1 44 28 PM](https://user-images.githubusercontent.com/16737484/116148313-9aa87000-a695-11eb-8111-0311194b901d.png)

cc: @himanshi-chhabra

Reply:
👍 Fine with me. thanks @SEAjamieD